### PR TITLE
fix vector's extend function

### PR DESCRIPTION
**Pull request by m-schen**
## What type of PR is this?

- [ ] API-change
- [x] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #

## What this PR does / why we need it:

### CODE REVIEW

**Comment by m-schen** [comment]
@matrix-meow rerun failed

**Comment by m-schen** [comment]
@matrix-meow rerun failed

File: pkg/container/vector/tools.go [hunk 1]
```diff
@@ -296,5 +296,7 @@ func (v *Vector) extend(rows int, m *mheap.Mheap) error {
 	}
 	// Setup v.Col
 	v.setupColFromData(0, tgtSz/v.GetType().TypeSize())
+	// extend the null map
+	nulls.TryExpand(v.Nsp, tgtSz)
```
Comment (fengttt) [inline pkg/container/vector/tools.go:300]: Please remove the XXX comment on top.

Second, should you only expand when original v.Nsp is not empty?   I am not sure about this ...

Comment (m-schen) [reply pkg/container/vector/tools.go:299]: yeah, I removed the comments.

and nulls.TryExpand has do the check work ( if Nsp is nil or not)

**Review by nnsgmsone — APPROVED** [review]

**Comment by m-schen** [comment]
@matrix-meow rerun failed

**Pull request merged**